### DISCUSSION TITLE
Curator overview throws "Could not find the image" errors when using an S3 disk

### DISCUSSION THREAD

**Discussion by Kovah** [awcodes/filament-curator#453] [Q&A] [answered]
Since I moved all my media files to an S3-compatible storage, the overview constantly throws "Could not find the image" errors for all the images loaded on the Curator overview page.

After digging deeper, it seems that on the overview page, Glide is used to display a thumbnail version of the image. However, I guess using a remote S3 storage is not compatible with Glide. (also see #220).

When looking at the MediaController, there is a switch for checking if a media is theoretically resizable (https://github.com/awcodes/filament-curator/blob/3.x/src/Http/Controllers/MediaController.php#L34), but it only takes into account some extensions (https://github.com/awcodes/filament-curator/blob/3.x/src/helpers.php#L11).

My question would be: is Glide able to resize images, even if they are stored on S3, or is this just a case not taken into account while checking the media on the controller? 🤔

**Comment by awcodes** [comment, marked as answer]
Glide works with remote images, but there will be some performance implications involved with that. Typically you would see that error if the permissions are off on the bucket because it's preventing glide from downloading the image.

I'm doing my best to improve the remote storage. But it's difficult to get right since everyone's setup can be different.

**Reply by Kovah** [reply]
My bad. 😞 After digging through the source code I noticed that there is a cloud_disk config. And I just forgot to add my disk to that config. 🤦
